app/index.jsx

```diff
@@ -88,16 +106,27 @@ class App extends Component {
             window.location.reload();
         }
     };
-
-
+    startingLanguage = 'en';
     render() {
+        const renderTopWidgets = () => (
+            <React.Fragment>
+                <ChartTitle />
+                <AssetInformation />
+                <ComparisonList />
+                <Notification
+                    messages={this.state.messages}
```

Comment:
I see the rationale behind this design decision. You wanted to maintain React's prop style of passing data. But it's also odd to have the logic of managing the notifications passed to the parent. It meant anyone who wants to use your `<Notification />` needs to have some version of `onMessage` and `onDestroyMessage`, wherever they use it.

Unless you have a better solution, I still think the `Notifier` design is better for this case.